### PR TITLE
default params in arrow functions pass in Chrome 49

### PR DESCRIPTION
**Pull request by adrianheine**
https://github.com/v8/v8/commit/ff932fe8f4f82a6832c4f649ad00b3ae59f53822 fixed default parameters in arrow functions, https://github.com/v8/v8/commit/e752f964b645fc7aafd99fd684914a10831619e8 removed the flag. The latter was released in V8 4.8.284, which was shipped in 49.0.2623. I did not check this, but it wasn't tested in 6c0b2fcfec89d97740ab42e7cf8eafe202ca1f65, either.

### CODE REVIEW

File: data-es6.js [hunk 1]
```diff
@@ -12960,7 +12960,7 @@ exports.tests = [
         typescript: true,
         traceur: true,
         edge15: false,
-        chrome50: true,
```
Comment (ljharb) [inline data-es6.js:12963]: can we add `chrome48: false` as well, assuming that's the case?

**Review by ljharb — CHANGES_REQUESTED** [review]
(please run the build process as well, to update the html)

File: data-es6.js (hunk 1 above)
Comment (adrianheine) [reply data-es6.js:12963]: I think it should be `chrome47: "flagged"`.

**Comment by adrianheine** [comment]
This doesn't affect the rendered HTML, since these environments are very obsolete.

File: data-es6.js (hunk 1 above)
Comment (ljharb) [reply data-es6.js:12963]: Perfect

**Review by ljharb — APPROVED** [review]

**Review by afmenez — APPROVED** [review]
LGTM

**Review by chicoxyzzy — APPROVED** [review]

**Pull request merged**